we-rent-app/src/components/ListingCard.tsx

```diff
@@ -0,0 +1,112 @@
+const ListingCard: React.FC<Props> = ({ listing, i }) => {
+    return (
+        <Grid item key={i} xs={3} sm={3} md={3}>
+            <Card elevation={12}>
+                <CardContent>
+                    <Stack direction="column" spacing={1}>
+                        <Box>
+                            <Typography variant="h4" color="textPrimary">
+                                ${listing.price}
+                            </Typography>
+                        </Box>
+                        <Box>
+                            <Typography
+                                variant="h6"
+                                color="textSecondary"
+                                sx={{ fontWeight: "bold" }}
+                            >
+                                {listing.address}
+                            </Typography>
+                        </Box>
+                        <Grid container spacing={1} alignItems="center">
+                            <Grid item>
+                                <Typography variant="body1">{listing.num_bedrooms}</Typography>
+                            </Grid>
+                            <Grid item>
+                                <KingBedTwoToneIcon />
+                            </Grid>
+                            <Divider
+                                orientation="vertical"
+                                variant="middle"
+                                flexItem
+                                color="gold"
+                            />
+                            <Grid item>
+                                <BathtubIcon />
+                            </Grid>
+                            <Divider
+                                orientation="vertical"
+                                variant="middle"
+                                flexItem
+                                color="gold"
+                            />
+                            <Grid item>
+                                <StraightenSharpIcon />
+                            </Grid>
+                            <Divider
+                                orientation="vertical"
+                                variant="middle"
+                                flexItem
+                                color="gold"
+                            />
+                        </Grid>
+                    </Stack>
+                </CardContent>
+                <CardActions>
+                    <Stack direction="row" spacing={1}>
+                        <Button>
+                            <FavoriteBorderSharpIcon />
+                        </Button>
+                        <Button>
+                            <ThumbDownOffAltSharpIcon />
+                        </Button>
+                        <Button>
+                            <BookmarkBorderSharpIcon />
+                        </Button>
+                        <Typography
+                            variant="h6"
+                            color="textPrimary"
```

Comment:
I think it looks good for now, we can revamp it later on. But could you make the number display in white text for now?